src/modules/control/include/control/control_node.h

```diff
@@ -51,6 +53,12 @@ class control_node
     int targetPointIndex;
     float least_distance;
     ros::Timer control_cmd_timer;
+    
+    //Heading Estimator
+    float prev_pos[2] = {0, 0};
```

Comment:
It doesn't matter because the estimator doesn't have good accuracy at static anyways, and the GPS will be running at 10 hz, which will update the heading estimator instantly as we start.